### ISSUE TITLE
Changes in this repo are not deployed to opencensus.io

### ISSUE THREAD

**Issue by songy23** [census-instrumentation/opencensus-website#713]
Previously once a PR is merged to this repo, the changes will show up on opencensus.io usually within a few minutes. However currently it looks like deployment to opencensus.io is broken. For example https://github.com/census-instrumentation/opencensus-website/pull/712 should have updated https://opencensus.io/integrations/sql/java_sql/ but it's showing the old content. Same for https://github.com/census-instrumentation/opencensus-website/pull/710 and https://opencensus.io/exporters/supported-exporters/go/stackdriver/.

/cc @rghetia @odeke-em

**Comment by odeke-em** [comment]
Interestingly Travis-CI shows that it built and deployed successfully to https://travis-ci.org/census-instrumentation/opencensus-website/builds/580447164

but the error seems to be that the Node.js version that is in our .travis.yml file as Node.js 7 is incompatible with Firebase CLI's expectations as per
```shell
Firebase CLI v7.3.0 is incompatible with Node.js v7.10.1 Please upgrade Node.js to version >= 8.0.0
```

**Comment by odeke-em** [comment]
I've mailed out https://github.com/census-instrumentation/opencensus-website/pull/714

**Comment by odeke-em** [comment]
The fix is now live https://opencensus.io/integrations/sql/java_sql/

**Comment by songy23** [comment]
There seems to be an issue with the banner...
![opencensus-io](https://user-images.githubusercontent.com/10536136/64298823-ac228e80-cf2b-11e9-819e-9240cb25471c.png)
I'm using Chrome 76.0. Will file a separate issue for that.
